protect_archiver/downloader/get_motion_event_list.py

```diff
@@ -16,7 +16,7 @@ def get_motion_event_list(
     session: Any, start: datetime, end: datetime, camera_list: List[Camera]
 ) -> List[MotionEvent]:
     motion_events_uri = (
-        f"{session.authority}{session.base_path}/events?type=motion"
+        f"{session.authority}{session.base_path}/events?type=ring,motion,smartDetectZone,smartDetectLine"
```

Comment:
Note the difference in the parameter - it used to be `type` and it is now `types`. See https://github.com/danielfernau/unifi-protect-video-downloader/issues/202#issue-1339884858

Reply:
Thanks for the hint. It's a work in progress and the Pull Request is still in Draft state. As soon as it's ready for testing I'll let you know.

First step was to check for all available and applicable event types on the API (for example, there's also stuff like 'deviceAdopted', 'videoExported', 'applicationUpdate' and so on) and compare that to your list in #202 . 

Next step on my end is to check if this API path is actually the current one or if it has been deprecated and is only kept for the next few versions to ensure backwards compatibility. Once all that is verified, I'll update the API path and its query keys.

Reply:
API path is still valid and current; added list of all available query parameters as inline documentation; ready for testing